# list the topics based on priority (lowest first)
twist_mux_topics: [cmd_move_base, cmd_joy]

# time for which lower priority topics are ignored if something of higher priority is published
hold_off_time: 1.0
topic_out_name: cmd_vel_mux
